bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/json/SafeExposureTime.java

```diff
@@ -56,10 +55,8 @@ public State getSafeExposure(int index) {
                 result = st6;
                 break;
             default:
-                result = BigInteger.ZERO;
-
+                result = null;
         }
-        return (result != BigInteger.ZERO && result != null) ? new QuantityType<>(result, SmartHomeUnits.MINUTE)
-                : UnDefType.NULL;
```

Comment:
More a general question, not recommendation: we also have `UnDefType.UNDEF`. What is the reasoning to chose one or the other?

Reply:
Well, this subject has been raised to me many times. I understood that UnDefType.UNDEF was only to be used before channels are somehow updated (persistence, binding...). IIRW taking a look at various bindings, this behaviour is not very well and clearly defined.

Reply:
Well, this subject has been raised to me many times. I understood that UnDefType.UNDEF was only to be used before channels are somehow updated (persistence, binding...). IIRW taking a look at various bindings, this behaviour is not very well and clearly defined.